### DISCUSSION TITLE
Deadline Target API Level

### DISCUSSION THREAD

**Discussion by William150722** [LawnchairLauncher/lawnchair#3382] [Q&A]
<img width="637" alt="Screenshot 2023-03-22 at 10 55 23 AM" src="https://user-images.githubusercontent.com/111726325/226790277-ce30f708-a539-4670-ba90-7529e3aac3b4.png">

Do we need to handle anything for this? Need target API level 33 and above.

**Comment by admin** [comment]
Hello. I know that this is a bit late, but this discussion seems to be unrelated to Lawnchair. As such, I'll close this.
